src/components/bulk-email-tool/bulk-email-form/test/BulkEmailForm.test.jsx

```diff
@@ -28,32 +30,24 @@ describe('bulk-email-form', () => {
     expect(warning).toBeTruthy();
   });
   test('Prevent form POST if invalid', async () => {
-    await act(async () => {
-      render(<BulkEmailForm courseId="test-course-id" />);
-      const submitButton = await screen.findByText('Submit');
-      fireEvent.click(submitButton);
-      const continueButton = await screen.findByText('Continue');
-      fireEvent.click(continueButton);
-      const recipientsInvalid = await screen.findByText('At least one recipient is required');
-      const subjectInvalid = await screen.findByText('A subject is required');
-      expect(recipientsInvalid).toBeTruthy();
-      expect(subjectInvalid).toBeTruthy();
-    });
+    render(<BulkEmailForm courseId="test-course-id" />);
+    fireEvent.click(screen.getByText('Submit'));
+    expect(await screen.findByRole('button', { name: /continue/i })).toBeInTheDocument();
```

Comment:
Any reason for the change from "toBeTruthy" to "toBeInTheDocument"?

Reply:
While looking into my testing issues, I found these expect extensions and decided to add them to our testing suite. Its a the same ends (in this case) but `toBeInTheDocument()` is more explicit, which I liked. Feel free to use it, but you don't need to change tests around for it.